test/fixtures/m3u8/absoluteUris.js

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Almost all the m3u8s and expected parses below are straight copies from videojs-contrib-hls. I'll call out the interesting additions.